/lib64/rc/sh/rc-cgroup.sh: line 89: printf: write error: No such device

ok this is the one that fails:
/sys/fs/cgroup/openrc/sshd/tasks
there is no sshd directory!

sys-apps/openrc-0.17 (/lib64/rc/sh/rc-cgroup.sh)

total 0
dr-xr-xr-x 11 root root   0 30.08.2015 20:49 ./
drwxr-xr-x 11 root root 220 30.08.2015 20:46 ../
drwxr-xr-x  2 root root   0 30.08.2015 20:49 cronie/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 dbus/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 fail2ban/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 gpm/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 lvmetad/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 metalog/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 net.enp0s3/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 udev/
drwxr-xr-x  2 root root   0 30.08.2015 20:49 virtualbox-guest-additions/
-rw-r--r--  1 root root   0 30.08.2015 20:49 cgroup.clone_children
-rw-r--r--  1 root root   0 30.08.2015 20:49 cgroup.procs
-r--r--r--  1 root root   0 30.08.2015 20:49 cgroup.sane_behavior
-rw-r--r--  1 root root   0 30.08.2015 20:46 notify_on_release
-rw-r--r--  1 root root   0 30.08.2015 20:49 release_agent
-rw-r--r--  1 root root   0 30.08.2015 20:47 tasks

and of course on next reboot it works without errors (seems random):
# ls -la /sys/fs/cgroup/openrc/sshd/
total 0
drwxr-xr-x  2 root root 0 30.08.2015 20:57 ./
dr-xr-xr-x 12 root root 0 30.08.2015 20:57 ../
-rw-r--r--  1 root root 0 30.08.2015 20:57 cgroup.clone_children
-rw-r--r--  1 root root 0 30.08.2015 20:57 cgroup.procs
-rw-r--r--  1 root root 0 30.08.2015 20:57 notify_on_release
-rw-r--r--  1 root root 0 30.08.2015 20:56 tasks

https://bugs.gentoo.org/show_bug.cgi?id=553978

--- ./rc-cgroup.sh	2015-08-29 22:35:15.448536778 +0300
+++ /lib64/rc/sh/rc-cgroup.sh	2015-08-29 22:43:11.222335345 +0300
@@ -86,7 +86,7 @@ cgroup_add_service()
 	if [ -d "$openrc_cgroup" ]; then
 		cgroup="$openrc_cgroup/$RC_SVCNAME"
 		mkdir -p "$cgroup"
-		[ -f "$cgroup/tasks" ] && printf "%d" 0 > "$cgroup/tasks"
+		[ -f "$cgroup/tasks" ] && printf "%d" 0 > "$cgroup/tasks" || echo "$cgroup/tasks" 1>&2
 	fi
 }
 
